code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm

```diff
@@ -47,7 +47,8 @@
 			name = "Natural " + name
 		if(data["boozepwr"])
 			boozepwr = data["boozepwr"]
-	addiction_types = list(/datum/addiction/alcohol = 0.05 * boozepwr)
+	if(boozepwr) // anything other than 0
```

Comment:
```suggestion
	if(boozepwr > 0)
```

save the comment by making it more explicit

Reply:
Before I commit that - you are aware that this will prevent the if block from running if `boozepwr` is negative, yes? That's the reason I chose `if(boozepwr)`.

Reply:
We do have drinks with negative bzpwr so this is right

Reply:
lol why can it be a negative, whatever